<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida"/>
        </item>
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Particulas</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="1">
            <widget class="QSpinBox" name="destinoX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_id">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_velocidad">
             <property name="text">
              <string>Velocidad:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="velocidad_lineEdit"/>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="origenY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QPushButton" name="agregarInicio_pushButton">
             <property name="text">
              <string>Agregar Inicio</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_origenX">
             <property name="text">
              <string>Origen en x:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="origenX_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Color</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="1">
               <widget class="QSpinBox" name="green_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_blue">
                <property name="text">
                 <string>Blue:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="blue_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_red">
                <property name="text">
                 <string>Red: </string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QSpinBox" name="red_spinBox">
                <property name="maximum">
                 <number>500</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_green">
                <property name="text">
                 <string>Green:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_destinoX">
             <property name="text">
              <string>Destino x:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_destinoY">
             <property name="text">
              <string>Destino y:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_origenY">
             <property name="text">
              <string>Origen en y:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="destinoY_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="id_lineEdit"/>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QPushButton" name="agregarFinal_pushButton">
             <property name="text">
              <string>Agregar Final</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="3">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Id particula</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="mostrarTabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Dibujar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="dibujar_pushButton">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="limpiar_pushButton">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuOrdenar">
    <property name="title">
     <string>Editar</string>
    </property>
    <widget class="QMenu" name="menuOrdenar_particulas_por">
     <property name="title">
      <string>Ordenar particulas por</string>
     </property>
     <addaction name="action_OrderByIdAsc"/>
     <addaction name="action_OrderByIdDes"/>
     <addaction name="separator"/>
     <addaction name="action_OrderByDistanciaAsc"/>
     <addaction name="action_OrderByDistanciaDes"/>
     <addaction name="separator"/>
     <addaction name="action_OrderByVelocidadAsc"/>
     <addaction name="action_OrderByVelocidadDes"/>
    </widget>
    <addaction name="menuOrdenar_particulas_por"/>
   </widget>
   <widget class="QMenu" name="menuVer">
    <property name="title">
     <string>Ver</string>
    </property>
    <addaction name="actionGrafo"/>
   </widget>
   <widget class="QMenu" name="menuAlgoritmos">
    <property name="title">
     <string>Algoritmos</string>
    </property>
    <addaction name="actionRecorrido_Profundidad_Amplitud"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuVer"/>
   <addaction name="menuOrdenar"/>
   <addaction name="menuAlgoritmos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPor_id">
   <property name="text">
    <string>Por id</string>
   </property>
  </action>
  <action name="actionPor_distancia">
   <property name="text">
    <string>Por distancia</string>
   </property>
  </action>
  <action name="action_OrderByIdAsc">
   <property name="text">
    <string>Id (ascendente)</string>
   </property>
  </action>
  <action name="action_OrderByDistanciaDes">
   <property name="text">
    <string>Distancia (descendente)</string>
   </property>
  </action>
  <action name="action_OrderByVelocidadAsc">
   <property name="text">
    <string>Velocidad (ascendente)</string>
   </property>
  </action>
  <action name="actionGrafo">
   <property name="text">
    <string>Grafo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="action_OrderByIdDes">
   <property name="text">
    <string>Id (descendente)</string>
   </property>
  </action>
  <action name="action_OrderByDistanciaAsc">
   <property name="text">
    <string>Distancia (ascendente)</string>
   </property>
  </action>
  <action name="actionRecorrido_Profundidad_Amplitud">
   <property name="text">
    <string>Recorrido Profundidad | Amplitud</string>
   </property>
  </action>
  <action name="action_OrderByVelocidadDes">
   <property name="text">
    <string>Velocidad (descendente)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
